daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/ValidationError.scala

```diff
@@ -378,43 +371,38 @@ final case class EModuleVersionDependencies(
   override def context: Context = Context.None
 }
 
-final case class EMissingInterfaceMethod(
+final case class EMissingMethodInInterfaceInstance(
     context: Context,
-    template: TypeConName,
-    iface: TypeConName,
     method: MethodName,
 ) extends ValidationError {
   override protected def prettyInternal: String =
-    s"Template $template is missing method '$method' in its implementation of interface $iface."
+    s"Interface instance lacks an implementation for method '$method'."
```

Comment:
This takes fewer arguments because all the relevant information is in `context` and `ValidationError.pretty` prepends that information to `prettyInternal`.